backend/dataall/db/connection.py

```diff
@@ -11,7 +11,7 @@
 from .. import db
 from ..db import Base
 from ..db.dbconfig import DbConfig
-from ..utils import Parameter, Secrets
```

Comment:
I guess usage should also be removed from [line 118](https://github.com/awslabs/aws-dataall/pull/550/files#diff-0674a6fd89dc34e30247435f9c6e4db20db0934e3171aa94f3acf22a592f14eaL118).

There is also a boto3 call in line 120 - if you want to outsource that as well.